Sources/BoxSDK.swift

```diff
@@ -469,7 +473,7 @@ extension BoxSDK {
     /// - Returns: Standard URL object to be used for authorization in external browser.
     public func makeAuthorizeURL(callbackURL: String? = nil, state: String? = nil) -> URL {
```

Comment:
Is the callbackURL parameter still required here now? I don't see how it is used in the SDK.

Reply:
Good point, I'll remove this callbackURL too. I couldn't find where it was used either, my only guess was that it might've been something that partially carried over from the objective-c sdk.